ngrinder-controller/src/main/java/org/ngrinder/perftest/service/PerfTestRunnable.java

```diff
@@ -229,6 +230,8 @@ public void doTest(final PerfTest perfTest) {
 			LOG.debug("Stack Trace is : ", ex);
 			doTerminate(perfTest, singleConsole, ex.getMessage());
 			notifyFinish(perfTest, StopReason.ERROR_WHILE_PREPARE);
+		} finally {
+			cleanUp(perfTest);
```

Comment:
Add fallback to cleanup resource.

1. delete dist folder.
2. delete hazelcast map data for sampling.